eng/packages/http-client-csharp/generator/TestProjects/CadlRanch.Tests/Infrastructure/ProcessTracker.cs

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace TestProjects.CadlRanch.Tests
+{
+    // Uses Windows Job Objects to ensure external processes are killed if the current process is terminated non-gracefully.
+    internal static class ProcessTracker
```

Comment:
Can we use the one from the Test Framework? https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core.TestFramework/src/ProcessTracker.cs